SpatialGDK/Documentation/known-issues.md

```diff
@@ -1,28 +1,13 @@
-| When launching a client from a cloud deployment, it chooses the first **.exe** file in alphabetical order, rather than selecting a particular one. | 2019-03-06 | [LAUNCH-341](https://improbableio.atlassian.net/browse/LAUNCH-341) | When building your client, an underscore is added to the correct executable name. |
-| NetMulticast RPCs do not work on Singleton Actors. | 2019-03-13 | [UNR-856](https://improbableio.atlassian.net/browse/UNR-856) | Do not use NetMulticast RPCs on Singleton Actors.|
-| Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, such as playing the timeline on a multicast event, or replicating what is changed by the timeline. |
-| Generating schema for `UClass`es that are not explicitly loaded/referenced by the currently loaded level is not supported. For example, a hard coded path loaded at run-time will not be detected by the schema generator. | 2019-03-25 |  | Reference the class explicitly from an already loaded class (Level blueprint, GameMode etc). Alternatively, you can open the Blueprints for soft references before generating schema so that they are loaded into memory. |
-| Level streaming might affect performance in certain scenarios. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
-| Changing some SpatialOS settings after launching a local deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration. |
-| The GDK ignores the `bHidden` specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Do not rely on `bHidden`, set `bReplicates` to the correct value instead. |  
+For all other Known Issues, please visit [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. Feel free to contribute and subscribe to any issue, and raise new ones from [this page](https://github.com/spatialos/UnrealGDK/issues).
```

Comment:
```suggestion
For all other known issues, please visit [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. Feel free to contribute and subscribe to any issue, and raise new ones from [this page](https://github.com/spatialos/UnrealGDK/issues).
```

Reply:
We avoid proper nouns where we can because it's a mare to maintain (see Unreal or Unity docs!).